src/Attestation/Attestation.Test/ScenarioTests/AttestationTestRunner.cs

```diff
@@ -86,7 +86,7 @@ private static AttestationClient GetAttestationClient(MockContext context)
                 var connectionInfo = new ConnectionString(Environment.GetEnvironmentVariable("TEST_CSM_ORGID_AUTHENTICATION"));
                 string servicePrincipal = connectionInfo.GetValue<string>(ConnectionStringKeys.ServicePrincipalKey);
                 string servicePrincipalSecret = connectionInfo.GetValue<string>(ConnectionStringKeys.ServicePrincipalSecretKey);
-                string aadTenant = connectionInfo.GetValue<string>(ConnectionStringKeys.AADTenantKey);
+                string aadTenant = connectionInfo.GetValue<string>(ConnectionStringKeys.TenantIdKey);
```

Comment:
Shall it be changed?

Reply:
AADTenantKey is ambiguous while TenantIdKey is more straightforward. It only involved a few changes. Recommend updating the value.